rails/bad-paths-can-trigger-invalid_cross_origin_request.md

```diff
@@ -0,0 +1,41 @@
+# Bad JS paths can trigger `InvalidCrossOriginRequest`
```

Comment:
Maybe pedantic, but can you shorten this title so it doesn't wrap to a second line? The exception class is already called out in the test failure, so I don't think it's required in the title.